backend/services/user_service.py

```
@@ -0,0 +1,39 @@
+
+
+def create_user(db: Session, user: UserCreate):
+    db_user = User(username=user.username, email=user.email,
+                   password=user.password, activate=user.activate)
+    db.add(db_user)
+    db.commit()
+    db.refresh(db_user)
+    return db_user
+
+
+def update_user(user_id: int, user_data: UserCreate, db: Session):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)